__tests__/lib/cli/format.js

```diff
@@ -35,6 +35,14 @@ describe( 'utils/cli/format', () => {
 				input: [ '{"json":broken json with spaces}' ],
 				expected: [ '"{"json":broken json with spaces}"' ],
 			},
+			{
+				input: [ '--foo=\'bar1 "bar2" "bar3"\'' ],
+				expected: [ '--foo=\'bar1 "bar2" "bar3"\'' ],
+			},
+			{
+				input: [ '--foo', '\'bar1 "bar2" "bar3"\'' ],
+				expected: [ '--foo', '\'bar1 "bar2" "bar3"\'' ],
+			},
```

Comment:
@chriszarate I don't think these two test cases are valid:

If the argument is `--foo=\'bar1 "bar2" "bar3"\'` then it actually (correctly) becomes three different arguments: `--foo='bar1` `bar2` `bar3'`

Same for the other one: `--foo \'bar1 "bar2" "bar3"\'` becomes `--foo` `'bar1` `bar2` `bar3'`

Reply:
Ok, can you add some valid test cases for the change you're making?

Reply:
Updated your tests instead: https://github.com/Automattic/vip-cli/pull/1626/commits/5941cca74c157a4b396e035c05b870546da9a6e8